superset/utils/core.py

```diff
@@ -1203,3 +1203,10 @@ class DatasourceName(NamedTuple):
 def get_stacktrace():
     if current_app.config.get("SHOW_STACKTRACE"):
         return traceback.format_exc()
+
+
+def merge_dicts(*args) -> dict:
+    target: Dict[Any, Any] = dict()
+    for d in args:
```

Comment:
should we use a None check here ?, 
I was testing this on my own version and turned out ```SupersetModelView.method_permission_name ``` did not existed because of some rebase error.

Thanks